ports/renesas-ra/Makefile

```diff
@@ -328,6 +349,11 @@ SRC_C += \
 	$(BOARD_DIR)/src/hal_entry.c \
 	$(wildcard $(BOARD_DIR)/*.c)
 
+ifeq ($(BOARD),$(filter $(BOARD),VK_RA4W1 VK_RA6M3 VK_RA6M5))
```

Comment:
Can you please filter on the series and in `machine_dac.c` use something like:
```
#if MICROPY_HW_ENABLE_DAC
machine_dac_init()...
#endif
```

This way other `RA6M5` boards can use this without editing the Makefile.

Reply:
Again DAC is not confirmed it works on all boards, so ot will remain ON for those it is tested with.

Reply:
It will not be on/initialized unless `MICROPY_HW_DAC0` or `MICROPY_HW_DAC1` is defined in `mpconfigboard.h` (Note you disabled by default in `mpconfigboard_common.h`), building the drivers in `ra/` should not affect boards that don't use it.